client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/buffer/BufferPacker.java

```diff
@@ -66,6 +69,11 @@ public void process(Buffer buffer, int subIdx) throws InterruptedException {
       cachedBuffer = buffer;
       int targetSubIdx = currentSubIdx;
       currentSubIdx = subIdx;
+      logger.debug(
```

Comment:
other pr has covered this file changes. so all the changes of bufferpacker in this pr can be removed

Reply:
OK. Restored.